### DISCUSSION TITLE
python wheel, how to draw multiple res to one fig

### DISCUSSION THREAD

**Discussion by sfhmzpig** [MSKCC-Epi-Bio/decisioncurveanalysis#22] [General]
How to plot the results of multiple models on one graph

**Comment by VickersA** [comment]
This is built in as a core part of the software, you just add all the models you want. In R, for instance, it is dca(endpoint ~ model1 + model2 + model3 etc. etc.; in Stata it is dca endpoint model1 model2 model3 etc.

**Comment by shaunporwal** [comment]
@sfhmzpig 

The Python tutorial on [decisioncurveanalysis.org](https://mskcc-epi-bio.github.io/decisioncurveanalysis/) has this example code you can use:

```python
dca_multi_df = \
    dca(
        data=df_cancer_dx,
        outcome='cancer',
        modelnames=['famhistory', 'cancerpredmarker'], # Here you can include multiple model column names 
        thresholds=np.arange(0,0.36,0.01)
    )

plot_graphs(
    plot_df=dca_multi_df,
    y_limits=[-0.05, 0.2],
    graph_type='net_benefit'
    )
```

For more context please check out the tutorial. If you're still having trouble let me know.
